ftp/src/test/scala/akka/stream/alpakka/ftp/CommonFtpSourceSpec.scala

```diff
@@ -90,9 +115,7 @@ trait CommonFtpSourceSpec extends BaseSpec {
       probe.expectComplete()
 
       val expectedNumOfBytes = getLoremIpsum.getBytes().length * numOfFiles
-      val total = result.foldLeft(0L) {
-        case (acc, next) => acc + next.count
-      }
+      val total = result.map(_.count).sum
```

Comment:
in a test readability definitely wins over pers in things like these IMHO, thanks!